### ISSUE TITLE
Excel 2013 Issues Warning and Functions Do Not Work

### ISSUE THREAD

**Issue by EToreo** [chuanyi/msexcel-builder#20]
When I open files I generate with msexcel-builder in Excel 2013 I get a very annoying message stating that it had a hard time converting the file and that some things may be messed up.  Indeed they are, the background color of cells is not correct and I am unable to use functions such a "sum" when I edit cells.

Updating this package to be compatible with the newer versions of Office would be greatly appreciated.  I really like this package, but I need it to work with the newer versions of Office.

**Comment by abhishek-g** [comment]
Yes Same issue here. Does not open on LibreOffice too. Only able to open in Google Sheets.
Greatly appreciate the upgradation as i love how easy it has made excel creation for us.
